src/views/SettingsPage.vue

```diff
@@ -73,6 +73,58 @@
             </div>
           </form>
         </section>
+        <section class="fields">
+          <h1>Add New User</h1>
+          <form @submit.prevent="addNewUser">
+            <div class="field field--half">
+              <label>
+                <span>Email Address</span>
+                <input
+                  v-model="pendingChanges.newEmailAddress"
+                  type="email"
+                  autocomplete="new-email"
```

Comment:
Remove `autocomplete` because the user using this function is creating profile for other people and we don't want the browser to automatically fill in the information of the "current user."